### PR TITLE
f3d 1.3.1

### PR DESCRIPTION
**Pull request by Meakk**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by Meakk** [comment]
Failing build on Linux is an old issue related to Assimp and `librt.so`, see: https://github.com/assimp/assimp/issues/4428  
I'm confused why it was working for f3d 1.2 though.  
Could it be related to these commits: e1cb36c524a960f64916ed2833778a3e761e3223 and 49f09ee86b97e7ed6ec4fd53380afda863f572c6 ?

**Comment by Meakk** [comment]
I investigated, and it seems like `librt.so` has been changed to `librt.so.1` and `librt.a` in recent Linux glibc versions.  
On Ubuntu 22.04 the change is effective, so `f3d` cannot link to `assimp` because it exposes through its CMake interface an indirect dependency to `librt.so` which does not exist in Ubuntu 22.04 (`assimp` bottle seems to have been [generated](https://github.com/Homebrew/homebrew-core/actions/runs/2417388638) with Ubuntu 16.04 as far as I can tell)  

Any idea @chenrui333?  
Is it possible to trigger the generation of a new bottle of `assimp` using Ubuntu 22.04?

**Comment by danielnachun** [comment]
I rebased this now that https://github.com/Homebrew/homebrew-core/pull/110149 is merged.  That should fix the problem with `assimp`.

**Comment by mwestphal** [comment]
Similar issue but with another lib:

```
2022-09-12T09:13:01.2944141Z make[2]: *** No rule to make target '/usr/lib/x86_64-linux-gnu/libdl.so', needed by 'lib/libf3d.so'.  Stop.
2022-09-12T09:13:01.2944491Z make[2]: *** Waiting for unfinished jobs....
```

**Comment by mwestphal** [comment]
Exactly the same issue, but with libc6-dev, where libdl.so became libdl.a.

Which dependency should be rebuild in your opinion @Meakk @danielnachun ?

**Comment by Meakk** [comment]
I took a look at my local installations of Alembic, Assimp, OCCT, and VTK.  
Only OCCT and VTK expose `dl` in their CMake targets, but none use absolute paths so it should be fine.

At this point I have no idea how to fix this problem.

**Comment by mwestphal** [comment]
VTK is in the list of bottle that should be rebuilded according to the list in https://github.com/Homebrew/homebrew-core/issues/110010, maybe this would fix it ?

File: Formula/f3d.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 class F3d < Formula
   desc "Fast and minimalist 3D viewer"
   homepage "https://f3d-app.github.io/f3d/"
-  url "https://github.com/f3d-app/f3d/archive/refs/tags/v1.2.1.tar.gz"
-  sha256 "0d72cc465af1adefdf71695481ceea95d4a94ee9e00125bc98c9f32b14ac2bf4"
+  url "https://github.com/f3d-app/f3d/archive/refs/tags/v1.3.1.tar.gz"
+  sha256 "653dc4044e14d0618c1d947a8ee85d2513e100b3fc24bd6e51830131a13e795d"
   license "BSD-3-Clause"
-  revision 4
+  revision 1
```
Comment (carlocab) [inline Formula/f3d.rb:7]:
```suggestion
```

**Review by carlocab — COMMENTED** [review]
3 installs in the past year on Linux. No point delaying this further. We can build it on Linux once the dependencies have been fixed.

I'm ok to merge this once we remove the `revision`.

**Review by carlocab — APPROVED** [review]

**Comment by danielnachun** [comment]
I'll test locally to see if rebuilding `vtk` and/or OCCT fixes the problem.  But the PR shouldn't wait for this.  It's possible there is an unrelated issue we haven't resolved yet and that could be more challenging to fix.

**Pull request closed without merging**